src/rule-must-colocate-fragment-spreads.js

```diff
@@ -155,10 +155,12 @@ function checkColocation(context) {
               loc: utils.getLoc(context, node, queriedFragments[fragment]),
               message:
                 `This spreads the fragment \`${fragment}\` but ` +
-                'this module does not use it directly. If a different module ' +
-                'needs this information, that module should directly define a ' +
-                'fragment querying for that data, colocated next to where the ' +
-                'data is used.\n'
+                `this module does not use it directly or it the fragment is named incorrectly. If a different module ` +
```

Comment:
I'd even remove the first part of the error message. Let's make it clear that the problem is a) name of the imported component and b) the underscore.